ros/api/v0/recommendations.py

```diff
@@ -56,11 +56,11 @@ def get(self, host_id):
                     for skey in rules_columns:
                         recommendation[skey] = eval("f'{}'".format(rule_dict[skey]))
                     recommendations_list.append(recommendation)
+        else:
+            recommendations_list = []
```

Comment:
No need of `else` block as `recommendations_list` already initialized to empty array.